docs/how_to_guides/record.md

```diff
@@ -444,7 +418,7 @@ updated_data = [
     }
     for sample in data
 ]
-dataset.records.update(records=updated_data)
+dataset.records.add(records=updated_data)
 ```
 !!! note "Update the metadata"
     To update the metadata of a record, you can iterate over the records and update the metadata dictionary by key or using `metadata.update`. After that, you should update the records in the dataset.
```

Comment:
`metadata.update` in this paragraph should be changed to `metadata.add` or `metadata.log` (depending on the final name). Same for the paragraph right below the "Update records" heading.